src/main/java/mate/academy/model/Book.java

```diff
@@ -20,18 +20,18 @@ public class Book {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
-    @Column(nullable = false, name = "title")
```

Comment:
There is no need to delete this configuration,
This will protect you from someone trying to directly write empty values to the database
Bring them all back